sdk/synapse/synapse-access-control-rest/package.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "@azure-rest/synapse-access-control",
+  "author": "Microsoft Corporation",
+  "description": "A generated Rest Client SDK for AccessControlClient.",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/synapse/synapse-access-control/README.md",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "sdk-type": "client",
+  "version": "1.0.0-beta.1",
+  "dependencies": {
+    "@azure-rest/core-client": "1.0.0-beta.7",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.3.0",
+    "@azure/core-paging": "^1.2.0",
+    "tslib": "^2.2.0"
+  },
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/accessControlClientContext.ts",
```

Comment:
does the rest client layer have this file?

Reply:
Good catch!